packages/eslint-plugin-pf-codemods/lib/rules/v5/table-rename-TableComposable.js

```diff
@@ -0,0 +1,86 @@
+const {getPackageImports} = require("../../helpers");
+
+// https://github.com/patternfly/patternfly-react/pull/8892
+module.exports = {
+  meta: {fixable: "code"},
+  create: function (context) {
+    const imports = getPackageImports(context, "@patternfly/react-table");
+    const tableComposableImport = imports.find(
+      (imp) => imp.imported.name === "TableComposable"
+    );
+
+    return imports.length === 0 || !tableComposableImport
+      ? {}
+      : {
+          // update opening/closing elements
+          JSXIdentifier(node) {
+            const nodeName = node.name;
+            const importedNode = imports.find(
+              (imp) => imp.local.name === nodeName
+            );
+            const parentNode = node.parent;
+            const isOpeningTag = parentNode.type === "JSXOpeningElement";
+            const openingTagAttributes = isOpeningTag
+              ? parentNode.attributes
+              : parentNode.parent.openingElement?.attributes;
+            const hasDataAttr =
+              openingTagAttributes &&
+              openingTagAttributes.filter(
+                (attr) => attr.name?.name === "data-codemods"
+              ).length;
+            const isAliased =
+              importedNode?.local?.name !== importedNode?.imported?.name;
+            if (importedNode?.imported?.name === "Table" && hasDataAttr) {
+              // if data-codemods attribute, do nothing
+              return;
+            } else if (importedNode?.imported?.name === "TableComposable") {
```

Comment:
Table && hasDataAttr doesn't seem to be necessary as we aren't going to do anything if `imported.name === 'Table'` anyway (only if "TableComposable").

I think we can just remove that check and the `hasDataAttr` with it.

this codemod will only affect TableComposable which will never have a codemod data attr as we always convert to table at the same time (so that if they run the codemods again it doesn't try to deprecate it after converting it)